tensorboard/webapp/metrics/views/main_view/card_grid_container.ts

```diff
@@ -168,6 +173,25 @@ export class CardGridContainer implements OnChanges, OnDestroy {
 
   onPageIndexChanged(newIndex: number) {
     this.pageIndex$.next(newIndex);
+
+    let ScrollingElement = this.cdkScrollable?.getElementRef().nativeElement;
+    if (ScrollingElement) {
+      let relativePosition =
+        ScrollingElement.scrollHeight - ScrollingElement.scrollTop;
+      // Clear call stack to allow dom update before updating scroll to keep
+      // relative position.
+      setTimeout(this.scrollToRelativePosition.bind(this, relativePosition), 0);
+    }
+  }
+
+  scrollToRelativePosition(relativePosition: number) {
```

Comment:
Is there any possibility to add tests for the new functionality? (and, likewise, a test to ensure it doesn't get applied to presses on the top row of next/previous buttons?)

Reply:
Looking into testing the new functionality.

Reply:
Tests added. PTAL.